testing/test_junitxml.py

```diff
@@ -523,7 +538,9 @@ def test_func(arg1):
                 assert 0
         """
         )
-        result, dom = run_and_parse(family=xunit_family)
+        result, dom = run_and_parse(
+            "-o", "junit_logging=system-out", family=xunit_family
```

Comment:
Yes, since the default value is still `no` but the behavior reflects what >no< means, that is there is nothing stored neither from log, system-out nor system-err by default.
That is why I had to set `junit_logging` explicitly to `system-out` so that later `sysout` variable is not `None`